packages/fxa-settings/src/pages/ResetPasswordRedesign/AccountRecoveryConfirmKey/index.test.tsx

```diff
@@ -0,0 +1,195 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { renderWithLocalizationProvider } from 'fxa-react/lib/test-utils/localizationProvider';
+import { Subject } from './mocks';
+import { screen, waitFor } from '@testing-library/react';
+import { MozServices } from '../../../lib/types';
+import userEvent from '@testing-library/user-event';
+import { MOCK_RECOVERY_KEY } from '../../mocks';
+import GleanMetrics from '../../../lib/glean';
+
+const mockVerifyRecoveryKey = jest.fn((recoveryKey: string) =>
```

Comment:
Warning about recoveryKey is declared but never used. You can remove it, or prefix with _ to get rid of warning.